multimodal/vl2l/src/mlperf_inference_multimodal_vl2l/cli.py

```diff
@@ -374,6 +377,11 @@ class Dataset(BaseModel):
         ),
     ] = None
 
+    split: Annotated[
+        Literal["train", "test"],
```

Comment:
Could you make the type to be a list? So that if we want to use a concatination of both splits (in the same order of the list), it'd be possible.